src/code/templates/fv-egg-sort-game.js

```diff
@@ -91,8 +91,8 @@ let animationEvents = {
       }},
       hatchDrakeInBasket: { id: 1, complete: false, animate: function() {
         const { scale } = _this.props,
-              leftOffset = -250,
-              topOffset = -40;
+              leftOffset = -255,
+              topOffset = -140;
```

Comment:
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.